ansible/roles/operator-pipeline/templates/openshift/tasks/generate-index.yml

```diff
@@ -55,10 +55,9 @@ spec:
       env:
         - name: STORAGE_DRIVER
           value: vfs
-      workingDir: $(workspaces.output.path)
       securityContext:
-        runAsUser: 1000
-        runAsGroup: 1000
+        privileged: true
+      workingDir: $(workspaces.output.path)
```

Comment:
Now that this is running as root, we can probably get rid of most of the permission tweaks in the previous step and replace the `tee -a` in this one with a regular stdout redirection.
The chmod in the `determine-index-format` step can also be dropped, I think.

Reply:
Fixed and tested on stage cluster.